src/name.rs

```diff
@@ -806,6 +813,44 @@ mod namespaces {
         assert_eq!(resolver.find(name, &buffer), Unknown(b"unknown".to_vec()));
     }
 
+    #[test]
+    fn undeclared_xml_prefix() {
+        let name = QName(b"xml:lang");
+
+        let resolver = NamespaceResolver::default();
+        let buffer = Vec::new();
+
+        assert_eq!(buffer, b"");
+        assert_eq!(
+            resolver.resolve(name, &buffer, true),
```

Comment:
```suggestion
            resolver.resolve(name, b"", true),
```
Here and everywhere.